packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -272,14 +274,75 @@ const gateway = new appmesh.VirtualGateway(this, 'gateway', {
     port: 8080,
     tls: {
       mode: appmesh.TlsMode.STRICT,
-      certificate: appmesh.TlsCertificate.file({
-        certificateChainPath: 'path/to/certChain',
-        privateKeyPath: 'path/to/privateKey',
-      }),
+      certificate: appmesh.TlsCertificate.file('path/to/certChain', 'path/to/privateKey'),
     },
   })],
   virtualGatewayName: 'gateway',
 });
+
+// A Virtual Gateway with listener TLS from a SDS provided certificate
+const gateway2 = new appmesh.VirtualGateway(this, 'gateway2', {
+  mesh: mesh,
+  listeners: [appmesh.VirtualGatewayListener.http2({
+    port: 8080,
+    tls: {
+      mode: appmesh.TlsMode.STRICT,
+      certificate: appmesh.TlsCertificate.sds('secrete_certificate'),
+    },
+  })],
+  virtualGatewayName: 'gateway2',
+});
+```
+
+## Adding mutual TLS authentication
+
+Mutual TLS authentication is an optional component of TLS that offers two-way peer authentication. 
+To enable mutual TLS authentication, 
+add `mutualTlsCertificate` property to TLS Client Policy and/or `mutualTlsCertificate` property to TLS Listener.
```

Comment:
I believe listeners use the `mutualTlsValidation` property.

Reply:
Addressed.